.github/workflows/docker-image.yml

```diff
@@ -6,6 +6,7 @@ on:
     # Pattern matched against refs/tags
     tags:        
       - '*'           # Push events to every tag not containing /
+    branches: [ multiplatform-build-fix-3 ]
```

Comment:
Maybe remove this field before merging

Reply:
that was for debug, to be sure it actually works on CI

Reply:
and it knda is https://hub.docker.com/layers/qdrant/qdrant/multiplatform-build-fix-3/images/sha256-9f848a87de8021292f66ef7145485237cdebf57f09dfeb5131b3e6befb8adb8a?context=repo

Reply:
That is useful for debugging this PR. But do we really need this in master branch?